test/Sentry.Extensions.Logging.Tests/LoggingTests.cs

```diff
@@ -55,6 +60,10 @@ public void Log_CapturesEvent(LogLevel logLevel)
     [InlineData(LogLevel.Trace)]
     public void Log_AddsBreadcrumb(LogLevel logLevel)
     {
+#if __IOS__
+        Skip.If(true, "Flaky on iOS");
```

Comment:
Any ticket or pointer we can link from here? Curious if we should try to fix it or what not

Reply:
No ticket... the tests in this class just failed consistently on iOS. All works fine on other platforms. 

I do think we should circle back and try to find the root causes of these failures eventually. I've created this ticket so that doesn't get lost or forgotten about:
- https://github.com/getsentry/sentry-dotnet/issues/3649